tests/v2/validation/Jenkinsfile

```diff
@@ -13,7 +13,7 @@ node {
     def testResultsJSON = "results.json"
     def envFile = ".env"
     def rancherConfig = "rancher_env.config"
-    def branch = "release/v2.8"
+    def branch = "release/v2.10"
```

Comment:
Do we always have the Jenkinsfile across all backports point to the same branch by default?

Reply:
afaik yes